platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -140,6 +142,13 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
+    @Override
+    public Collection<Describable> getInfoForOptionalMigratables() {
+        return dataMigratables.stream()
```

Comment:
@coyotesqrl and some others in #developers recommend we get used to streaming syntax. Also IntelliJ recommended it.